ntbs-service/wwwroot/css/_colors.scss

```diff
@@ -8,6 +8,8 @@ $grey: #999999;
 $lighter-grey: #CCCCCC;
 $shaded-background-color: #F0F4F5;
 
+$phe-navy: #002776;

```

Comment:
Where did you find this colour?

Reply:
This was being used on line ~237 of `ntbs-service/wwwroot/css/site.scss` to provide the heading colour of the previous panel. I asked Nancy about it, and she told me it comes from the PHE colour palette (page 14 of the PHE Identity Policy, which is stored in Sharepoint).